ibm_catalog.json

```diff
@@ -48,6 +48,47 @@
                 }
               ]
             },
+            "dependency_version_2": true,
+            "dependencies": [
+              {
+                "flavors": [
+                  "fully-configurable"
+                ],
+                "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+                "id": "045c1169-d15a-4046-ae81-aa3d3348421f-global",
+                "name": "deploy-arch-ibm-apprapp",
+                "version": "v1.5.0",
```

Comment:
just a note that we will lock into a newer version once https://github.com/terraform-ibm-modules/terraform-ibm-app-configuration/pull/220 is merged